test/ecto/changeset_test.exs

```diff
@@ -723,6 +724,13 @@ defmodule Ecto.ChangesetTest do
     assert changeset.changes == %{}
     assert changeset.errors == [title: {"is blank", [validation: :required]}, body: {"is blank", [validation: :required]}]
 
+    # When type is binary skip value triming (byte 12 == emtpy character in strings)
+    changeset =
+        changeset(%{color: <<12, 12, 12>>})
```

Comment:
Alignment here is a bit weird. 

Maybe a oneliner could work here `validate_required(changeset(%{color: <<12, 12, 12>>}), :color)`

Reply:
Thanks for the feedback, I changed the format to a oneliner but used the same format as in the tests above (see test when missing).